src/SDKs/Automation/Automation.Tests/ScenarioTests/AutomationTests.cs

```diff
@@ -16,7 +16,7 @@ namespace Automation.Tests.ScenarioTests
 
     public class AutomationTest 
     {
-        [Fact]
+        //[Fact]
```

Comment:
Mark as a skipped tests instead of commenting

Reply:
Fixed.